iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/conf/CommonDescriptor.java

```diff
@@ -266,6 +266,11 @@ private void loadPipeProps(Properties properties) {
             properties.getProperty(
                 "pipe_hardlink_wal_enabled",
                 Boolean.toString(config.getPipeHardLinkWALEnabled()))));
+    config.setPipeFileReceiverFsyncEnabled(
+        Boolean.parseBoolean(
+            properties.getProperty(
+                "pipe_file_receiver_fsync_enabled",
```

Comment:
Should this also be added to the configuration file?

Reply:
This kind of parameter adjustment may bring problem of correctness, so it is better to treat it as a secret parameter. It is deliberately not placed in the configuration file.